### DISCUSSION TITLE
Vindriktning not visible in console/no data

### DISCUSSION THREAD

**Discussion by MatzeG** [arendst/Tasmota#16576] [Q&A]
hello 
i am trying the replica ikea vindriktning with tasmota extension. unfortunately the tasmota does not recognize the vindriktning sensor. the BMP180 with temperature and air pressure is recognized and also provides suitable data. on the tasmota runs a firmware all-sensors and in the configuration i have set D1 GPIO5 to vindriktning. 

Anyone have an idea what I can do?

**Comment by barbudor** [comment]
You should use RX GPIO3 for the Vindriktning in order to use hardware serial
Beside that, without more details such as your detailled configuration, boot logs with Weblog 3, ... it's difficult to guess what you could have done wrong

**Comment by MatzeG** [comment]
Thanks for the help. I think I probably destroyed the connector with the soldering iron. I'll get me times another Vindriktning.
